### PR TITLE
Cherrypick : Check for Default values on Global Registration Template

### PR DESCRIPTION
**Pull request by adarshdubey-star**
### Problem Statement
Check for Default values on Global Registration Template

### CODE REVIEW

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_registration.py -k test_positive_verify_default_values_for_global_registration
airgun: 1087

File: tests/foreman/ui/test_registration.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    )
+    with module_target_sat.ui_session() as session:
+        cmd = session.host.get_register_command(
+            {'general.activation_keys': ak.name, 'general.operating_system': default_os.title},
+            full_read=True,
+        )
+    assert cmd['general']['organization'] == 'Default Organization'
+    assert cmd['general']['location'] == 'Default Location'
+    assert cmd['general']['capsule'] == 'Nothing to select.'
+    assert cmd['general']['operating_system'] == ''
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_registration.py:50]:
```suggestion
    assert cmd['general']['operating_system'] == default_os.title
     assert cmd['general']['activation_keys'] == ak.name
```

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_registration.py -k test_positive_verify_default_values_for_global_registration
airgun: 1087

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_registration.py -k test_positive_verify_default_values_for_global_registration

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_registration.py -k test_positive_verify_default_values_for_global_registration
airgun: 1095

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**